arelle/RuntimeOptions.py

```diff
@@ -11,7 +12,7 @@
 from arelle.typing import TypeGetText
 from arelle.ValidateDuplicateFacts import DeduplicationType
 
-_: TypeGetText
+_: TypeGetText = gettext.gettext
```

Comment:
This overrides the gettext method for this module that's installed in the environment with the arelle domain and translations directory by the CLI and GUI. I'm assuming this is needed because the Python API doesn't run the install code paths for the [CLI](https://github.com/Arelle/Arelle/blob/78d321946eb754bda8651e4679d2002622290d79/arelle/CntlrCmdLine.py#L57) or [GUI](https://github.com/Arelle/Arelle/blob/78d321946eb754bda8651e4679d2002622290d79/arelle/Cntlr.py#L315)?

Reply:
Discussed separately. Some planning is required to get the API to a place where translations can be used prior to a `Cntlr` setting up translations *without* a `gettext.install('arelle')` call. For now, I'm proposing simply not attempting translations in `RuntimeOptions` since it's potentially the only place that might actually try to execute a translation prior to the `Cntlr` setting up translations.